app/models/katello/erratum.rb

```diff
@@ -46,10 +46,8 @@ def self.repository_association_class
     end
 
     def self.applicable_to_hosts(hosts)
-      self.joins(:content_facet_errata).joins(:content_facets).
-         where("#{Katello::Host::ContentFacet.table_name}.host_id" => hosts).
-         select("DISTINCT ON (#{self.table_name}.updated, #{self.table_name}.id) #{self.table_name}.*").
-         order("#{self.table_name}.updated desc, #{self.table_name}.id")
+      self.joins(:content_facets).
+        where("#{Katello::Host::ContentFacet.table_name}.host_id" => hosts).uniq
```

Comment:
Is there a test for this method? Something that would have or could in the future help protect against the SQL error before making it into the code base?